.gitmodules

```diff
@@ -47,3 +47,6 @@
 [submodule "meta-ros"]
 	path = meta-ros
 	url = https://github.com/bmwcarit/meta-ros
+[submodule "openembedded-core"]
+	path = openembedded-core
+	url = git://git.openembedded.org/openembedded-core
```

Comment:
This commit is useless, the layer is already present as `meta-openembedded`.

See https://www.yoctoproject.org/tools-resources/projects/openembedded-core and https://stackoverflow.com/questions/35904769/what-are-the-differences-between-open-embedded-core-and-meta-openembedded